rmg.py

```diff
@@ -35,8 +35,14 @@
 import logging
 
 # Before importing any RMG modules, check Python version
-import utilities
-utilities.check_python()
+try:
+    import utilities
+    utilities.check_python()
+
+except ImportError:  # This is likely the binary install version, which does not include utilities
+    # If this is in fact the binary version, conda has ensured that the python version is correct.
+    # It is okay to skip this test here
+    pass
```

Comment:
Perhaps we should put `utilities.check_python()` in the else clause?

Reply:
I agree, this is safer. I have made the change, do you want me to push the change now?